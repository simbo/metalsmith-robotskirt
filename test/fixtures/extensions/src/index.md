
http://domain.tld

https://domain.tld

ftp://domain.tld

user@domain.tld

www.domain.tld

```
fenced code
```

~~~
more fenced code
~~~

I am text.
<div>I am laxed spaced HTML</div>
I am more text.

This is _emphased_, but not inside var names like some_foo_bar.

#this is no header

Text can ~~not~~ be striked through.

E = mc^2

| Left-Aligned  | Center Aligned  | Right Aligned |
| :------------ |:---------------:| -----:|
| col 3 is      | some wordy text | $1600 |
| col 2 is      | centered        |   $12 |
