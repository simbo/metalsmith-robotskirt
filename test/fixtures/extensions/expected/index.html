<p><a href="http://domain.tld">http://domain.tld</a></p>

<p><a href="https://domain.tld">https://domain.tld</a></p>

<p><a href="ftp://domain.tld">ftp://domain.tld</a></p>

<p><a href="mailto:user@domain.tld">user@domain.tld</a></p>

<p><a href="http://www.domain.tld">www.domain.tld</a></p>
<pre><code>fenced <span class="hljs-tag">code</span>
</code></pre><pre><code>more fenced <span class="hljs-tag">code</span>
</code></pre>
<p>I am text.</p>

<div>I am laxed spaced HTML</div>

<p>I am more text.</p>

<p>This is <em>emphased</em>, but not inside var names like some_foo_bar.</p>

<p>#this is no header</p>

<p>Text can <del>not</del> be striked through.</p>

<p>E = mc<sup>2</sup></p>

<table><thead>
<tr>
<th align="left">Left-Aligned</th>
<th align="center">Center Aligned</th>
<th align="right">Right Aligned</th>
</tr>
</thead><tbody>
<tr>
<td align="left">col 3 is</td>
<td align="center">some wordy text</td>
<td align="right">$1600</td>
</tr>
<tr>
<td align="left">col 2 is</td>
<td align="center">centered</td>
<td align="right">$12</td>
</tr>
</tbody></table>
